conftest.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        '--quick', action='store_true', help='Use existing input files in tests/data'
+    )
+    parser.addoption(
+        '--fulldb', action='store_true', help='Test the fully-built conceptnet database'
+    )
+
+
+# https://docs.pytest.org/en/latest/example/simple.html#control-skipping-of-tests-according-to-command-line-option
+def pytest_configure(config):
+    config.addinivalue_line("markers", "requires_full_build: test requires the actual database to be built")
```

Comment:
Super nitpick: single quotes.

Reply:
The Daylight style guide isn't our style guide.

Reply:
I don't think we should spend any time caring about which kind of quotes we use, especially in copied code.